packages/extras/src/lib/components/Grid/Grid.svelte.d.ts

```diff
@@ -10,11 +10,35 @@ export type GridProps = Props<Mesh> & {
   sectionColor?: ColorRepresentation
   sectionSize?: number
   sectionThickness?: number
+  backgroundColor?: ColorRepresentation
+  backgroundOpacity?: number
   gridSize?: number | [number, number]
   followCamera?: boolean
   infiniteGrid?: boolean
   fadeDistance?: number
   fadeStrength?: number
+  type?: GridParams | TopologyParams | CircularParams | PolarParams
```

Comment:
Hm, this might look a bit odd:

```
<Grid type={{ type: 'lines' }} />
```

and I would prefer 

```
<Grid type="lines" />
```

and based on the prop `type` other props are necessary. Possible type implementation:


```ts
export type GridProps = Props<Mesh> & {
    axes?: 'xzy' | 'xyz' | 'zyx'
    // …
  } & (
      | {}
      | {
          type: 'grid'
        }
      | {
          type: 'lines'
          axis?: 'x' | 'y' | 'z'
        }
    )
```

Reply:
That being said, maybe I would rename `axes` to `plane`:

```
<Grid plane="xz" type="lines" axis="x" />
```

Reply:
Updated the types. We decided to do the `axes` to `plane` rename at next major bump.